Cargo.toml

```diff
@@ -32,7 +32,7 @@ cw-orch-fns-derive = { path = "packages/cw-orch-fns-derive", version = "0.18.0"
 cw-orch-networks = { path = "packages/cw-orch-networks", version = "0.20.0" }
 
 thiserror = { version = "1.0.21" }
-sha256 = { version = "1" }
+sha2 = { version = "0.10.8" }
```

Comment:
For using sha2 instead of sha256 - it's lighter(sha2 is dependency of sha256) and returns bytes, instead of String, which is easier to use in pair with HexBinary